### ISSUE TITLE
Add insertWithOnConflict() to the StorIOSQLite

### ISSUE THREAD

**Issue by artem-zinnatullin** [pushtorefresh/storio#521]
```java
storIOSQLite
  .internal()
  .insertWithOnConflict(InsertQuery, conflictAlgorithm, contentValues)
```

**Comment by Rainer-Lang** [comment]
Great! :+1:

**Comment by artem-zinnatullin** [comment]
@Rainer-Lang just released 1.5.0 with this feature! Should be available in 10-20 minutes
